src/engine/enginechannel.h

```diff
@@ -39,7 +39,8 @@ class EngineChannel : public EngineObject {
     };
 
     EngineChannel(const ChannelHandleAndGroup& handle_group,
-                  ChannelOrientation defaultOrientation = CENTER);
+                  ChannelOrientation defaultOrientation = CENTER,
+                  bool isInput = false);
```

Comment:
Ok, just change this to 
bool isTalkoverChannel = false
like you did in the cpp file and I am fine.